### PR TITLE
Django integration pipeline.

### PR DESCRIPTION
**Pull request by apollo13**
Preliminary work for #489.

### CODE REVIEW

**Comment by apollo13** [comment]
@felixxm Do we have a good set of tests that we want to run for psycopg regression testing? I checked but the Django testsuite doesn't seem to make use of any tags that we could use…

File: .github/workflows/3rd-party-tests.yml [hunk 1]
```diff
@@ -4,7 +4,9 @@ on:
   push:
     branches:
       - "master"
+      # TODO: What are the usecases for running on branches like this?
```
Comment (dvarrazzo) [inline .github/workflows/3rd-party-tests.yml:7]: Probably to test the pipeline themselves before pushing to master

**Comment by felixxm** [comment]
> @felixxm Do we have a good set of tests that we want to run for psycopg regression testing? I checked but the Django testsuite doesn't seem to make use of any tags that we could use…

Good question :thinking: I'd start from `./runtests.py postgres_tests backends queries`.

**Comment by apollo13** [comment]
Hi @dvarrazzo, I think we have a minimal set for tests now. Let me know what you think and then we can incrementally add new tests.

**Comment by dvarrazzo** [comment]
Hello @apollo13! This looks good from our point of view, thank you very much!

**Pull request merged**